scipy/interpolate/tests/test_polyint.py

```diff
@@ -699,10 +699,12 @@ def test_incorrect_inputs(self):
         y3 = [1, 2, 3]
         x1 = [1]
         y1 = [1]
+        y_empty = np.empty(shape=(4, 0))
 
         assert_raises(ValueError, CubicSpline, xc, y)
         assert_raises(ValueError, CubicSpline, xn, y)
         assert_raises(ValueError, CubicSpline, x, yn)
+        assert_raises(ValueError, CubicSpline, x, y_empty)
```

Comment:
Appreciate that the others do not do this, but for new additions it would be nice to check the message so that is certain that the right error is being raised. 

```suggestion
        with assert_raises(ValueError, match="`y` cannot be an empty array."):
            CubicSpline(x, y_empty)
```

Reply:
Yeah. That bit in the first try b/c I was outputting `ValueError`, just not from the right place. Thanks!